criteria/findability.md

```diff
@@ -0,0 +1,51 @@
+# Findability
+
+## Requirements
+
+* The codebase MUST be findable using a search engine by description of the problem it solves in natural language.
+* The codebase MUST be findable using a search engine by codebase name.
+* Maintainers SHOULD submit the codebase to relevant software catalogs.
+* The codebase SHOULD have a website which describes the problem the codebase solves using the preferred jargon of different potential users of the codebase (including technologists, policy experts and managers).
+* The codebase SHOULD have a unique and persistent identifier, such as a [Wikidata item](https://www.wikidata.org/wiki/Wikidata:Introduction) or [FSF software directory listing](https://directory.fsf.org/wiki/Main_Page), mentioning the major contributors, repository location and website.
```

Comment:
I'm not opposed to this, but at first glance from both websites it's not immediately clear to me what I have to do for this. So, this requires an investment for codebases to familiarize themselves with these concepts, and the benefit is not immediately clear to me why a codebase should have this.

Reply:
I agree with you here that this is a bit opaque. I think we need to add a sentence in "Why this is important" so that the value becomes clearer. Perhaps something like:

> - By having well-written metadata and being part of the semantic web, it will be easier for other people to refer, cite, disambiguate and discover the codebase through third party tools.

And I see here that we again really could use a secondary resource with good examples of this. We should perhaps raise the priority for creating it.

Reply:
I would say this is a criterion for the catalog owners instead of the codebase maintainers.